streams/za.m3u

```diff
@@ -30,28 +30,22 @@ https://webstreaming-8.viewmedia.tv/web_119/Stream/playlist.m3u8
 #EXTINF:-1 tvg-id="ROVTV.za",ROV TV (576p)
 https://webstreaming.viewmedia.tv/web_012/Stream/playlist.m3u8
 #EXTINF:-1 tvg-id="SABC1.za",SABC 1 (720p)
-http://185.59.221.131:8081/live/sabc1/playlist.m3u8
+https://sabconeta.cdn.mangomolo.com/sabc1/smil:sabc1.stream.smil/chunklist_b1600000_t64NzIwcA==.m3u8
+#EXTINF:-1 tvg-id="SABC2.za" tvg-logo="https://i.ibb.co/tmJGg7W/s2.jpg",SABC 2
+https://sabctwota.cdn.mangomolo.com/sabc2/smil:sabc2.stream.smil/chunklist_b1600000_t64NzIwcA==.m3u8
+#EXTINF:-1 tvg-id="SABC3.za" tvg-logo="https://i.ibb.co/PW7v0jg/s3.jpg",SABC 3
+https://sabctreta.cdn.mangomolo.com/sabc3/smil:sabc3.stream.smil/chunklist_b1600000_t64NzIwcA==.m3u8
+#EXTINF:-1 tvg-id="SABClehae.za" tvg-logo="https://i.ibb.co/WWsj2Vx/slahea.jpg",SABC Lehae
+https://sabctretalh.cdn.mangomolo.com/lehae/smil:lehae.stream.smil/chunklist_b2000000_t64NzIwcA==.m3u8
 #EXTINF:-1 tvg-id="SABCNews.za",SABC News (720p)
-http://185.59.221.131:8081/live/sabcnews/playlist.m3u8
+https://sabconetanw.cdn.mangomolo.com/news/smil:news.stream.smil/chunklist_b1600000_t64NzIwcA==.m3u8
 #EXTINF:-1 tvg-id="SeraphimTV.za",Seraphim TV [Not 24/7]
 https://restream.churchtv247.co.za/Apostle/Hggc@24/1.m3u8
 #EXTINF:-1 tvg-id="SowetoTV.za",Soweto TV (576p)
 https://cdn.freevisiontv.co.za/sttv/smil:soweto.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="TBNAfrica.za",TBN Africa (1080p)
 https://tbn-jw.cdn.vustreams.com/live/tbn-africa/live.isml/cc850928-d0d8-4336-84d0-999581129465.m3u8
-#EXTINF:-1 tvg-id="CapeTownTV.za",Cape Town TV
-https://cdn.freevisiontv.co.za/sttv/smil:ctv.stream.smil/chunklist.m3u8
```

Comment:
@telanus freevision works for me in VLC, is it not for you?
![image](https://github.com/user-attachments/assets/a392f409-db29-40d3-ade7-783a1fc80679)

Reply:
It's listed twice, I removed the last one, as they are both the same stream. There was a few "repeated" entries..